### DISCUSSION TITLE
Planning for the upcoming month?

### DISCUSSION THREAD

**Discussion by bjohnson-va** [firefly-iii/firefly-iii#6841] [Firefly III]
Hello, I'm transitioning away from a combo of spreadsheets + mint and, now that I've got my accounts and transactions to auto-import, this project is really looking promising for my use-case.

One thing that I had with my spreadsheet (but still not with Mint) is the ability to plan the upcoming month's budget ahead of time.

Is there a good way to do this in Firefly III?  I know there is the "auto-budget" thing, but it seems I need to wait until the first of the month for that to run, so it's hard to tell if my budget will be balanced.

Any advice?

**Comment by JC5** [comment]
You could always set the budgets by hand yourself, but otherwise no, there isn't really a good way to plan this ahead of time. I generally do this on the first of the month anyway.
